### PR TITLE
Remove dbt dependencies to enable more dbt versions

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

File: .github/workflows/fal_workflow.yml [hunk 1]
```diff
@@ -1,24 +1,30 @@
 name: Run fal scripts
 
 on:
-  push:
-    branches: [release]
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
```
Comment (mkam039) [inline .github/workflows/fal_workflow.yml:5]: should we run it on every PR? Might get expensive in GCP.

**Review by mkam039 — COMMENTED** [review]
Not sure if this should be run on every PR

File: .github/workflows/fal_workflow.yml (hunk 1 above)
Comment (chamini2) [reply .github/workflows/fal_workflow.yml:5]: What no workflow at all? I got confused with fal_dbt_test.

Comment (chamini2) [reply .github/workflows/fal_workflow.yml:5]: Or just on dispatch?

Comment (mkam039) [reply .github/workflows/fal_workflow.yml:5]: Just dispatch should be good. This is an example of incorporating fal into CI/CD. We're going to have a blog post about this.

**Comment by chamini2** [comment]
Let's wait for https://github.com/fal-ai/fal/pull/74 to test this is working

**Pull request merged**